### PR TITLE
fix: [DVC-8210] update JS SDKs to new SDK interfaces

### PR DESCRIPTION
**Pull request by jonathannorris**

### CODE REVIEW

File: proxies/nodejs/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "@devcycle/nodejs-server-sdk": "latest",
+    "@devcycle/nodejs-server-sdk": "*",
```
Comment (chris-hoefgen) [inline proxies/nodejs/package.json:3]: Just curious, what is the purpose of this change?

**Review by chris-hoefgen — APPROVED** [review]
LGTM.

File: proxies/nodejs/package.json (hunk 1 above)
Comment (jonathannorris) [reply proxies/nodejs/package.json:3]: I didn't notice that the package was getting updated when it was set to `latest`, which should be the same thing as setting it to `*`.
